config/consensus.go

```diff
@@ -290,6 +290,9 @@ type ConsensusParams struct {
 	// should the number of inner transactions be pooled across group?
 	EnableInnerTransactionPooling bool
 
+	// provide greater isolation for clear state programs
+	IsolateClearState bool
```

Comment:
IsolateClearState is actually go to change the semantics of *previous* TEAL versions too - just barely.  We've concluded this is a good change, as the previous semantics allow a malicious clear state to consume some extra resources. Still, it's not related to AVM version, but rather consensus, so it will go into effect if/when this consensus version is accepted by the network, even for old code (but not for old blocks, of course).

Reply:
@jannotti Got it - makes sense.  Thanks for the explanation.  Resolving.